packages/portalnetwork/src/client/client.ts

```diff
@@ -577,18 +612,31 @@ export class PortalNetwork extends (EventEmitter as { new (): PortalNetworkEvent
 
   private onTalkResp = (src: INodeAddress, sourceId: ENR | null, message: ITalkRespMessage) => {
     const srcId = src.nodeId
-    this.logger(`TALKRESPONSE message received from ${srcId}, ${message.response.toString()}`)
+    this.logger(`TALKRESPONSE message received from ${srcId}`)
   }
 
-  private handleStreamedContent(rcvId: number, content: Uint8Array) {
+  private handleStreamedContent(
+    rcvId: number,
+    content: Uint8Array,
+    contentType: HistoryNetworkContentTypes,
+    blockHash: Uint8Array
+  ) {
     this.logger(`received all content for ${rcvId}`)
-    const header = BlockHeader.fromRLPSerializedHeader(Buffer.from(content))
-    this.addContentToHistory(
-      1,
-      HistoryNetworkContentTypes.BlockHeader,
-      toHexString(header.hash()),
-      content
-    )
+    if (contentType === HistoryNetworkContentTypes.BlockHeader) {
+      this.addContentToHistory(
+        1,
```

Comment:
```suggestion
        0,
```
Content type 1 is blockbody